frontend/packages/core/src/chip.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import type { ChipProps as MuiChipProps } from "@material-ui/core";
+import { Chip as MuiChip } from "@material-ui/core";
+
+type variants = "error" | "unknown" | "pending" | "running" | "succeeded";
+export interface ChipProps extends Pick<MuiChipProps, "label"> {
+  variant: variants;
+}
+
+const CHIP_COLOR_MAP = {
+  error: "#FF8A80",
+  unknown: "#FFCC80",
+  pending: "#FFF59D",
+  running: "#C2C8F2",
+  succeeded: "#69F0AE",
+};
+
+const StyledChip = styled(MuiChip)({
+  height: "20px",
+  fontSize: "12px",
+  lineHeight: "20px",
+  cursor: "inherit",
+  margin: "10px",
+});
```

Comment:
```suggestion
const StyledChip = styled(MuiChip)(
  {
    height: "20px",
    fontSize: "12px",
    lineHeight: "20px",
    cursor: "inherit",
    margin: "10px",
  },
  props => ({
    background: CHIP_COLOR_MAP["data-variant'],
  })
);
```

Reply:
When I tried doing it like this in storybook it doesn't pass the data-variant.  As in, it will always be the default color rather than the color we want.